src/config/configuredChains.ts

```diff
@@ -3,14 +3,14 @@ import Telos from 'src/config/chains/telos';
 import UX from 'src/config/chains/ux';
 import Wax from 'src/config/chains/wax';
 
-import TelosTestnet from 'src/config/chains/telos-testnet';
+import PangeaTestnet from 'src/config/chains/pangea-testnet';
```

Comment:
also keep the old line (This is so that, when the time comes, we can merge back to their repository and they will accept the change) for backwards compatibility
`import TelosTestnet from 'src/config/chains/telos-testnet';`